analysis/topeft_run2/make_cr_and_sr_plots.py

```diff
@@ -519,17 +549,30 @@ def make_cr_fig(h_mc,h_data,unit_norm_bool,axis='process',var='lj0pt',bins=[],gr
             years[name] = [axis_name]
     hep.style.use("CMS")
     plt.sca(ax)
-    hep.cms.label(lumi='138')
+    if unblind: hep.cms.label(data=True, label='Preliminary',lumi='138')
+    else: hep.cms.label(lumi='138')
     # Hack for grouping until fixed
     grouping = {proc: [good_proc for good_proc in group[proc] if good_proc in h_mc.axes['process']] for proc in group if any(p in h_mc.axes['process'] for p in group[proc])}
+    #handle grouping for sumw2 hist separately (probably redundant)
+    grouping_sumw2 = {proc: [good_proc for good_proc in group[proc] if good_proc in h_mc_sumw2.axes['process']] for proc in group if any(p in h_mc_sumw2.axes['process'] for p in group[proc])}
     if group:
         vals = [h_mc[{'process': grouping[proc]}][{'process': sum}].eval({})[()][1:-1] for proc in grouping]
+        sumw2 = [h_mc_sumw2[{'process': grouping_sumw2[proc]}][{'process': sum}].eval({})[()][1:-1] for proc in grouping_sumw2]
         mc_vals = {proc: h_mc[{'process': grouping[proc]}][{'process': sum}].as_hist({}).values(flow=True)[1:] for proc in grouping}
+        mc_sumw2 = {proc: h_mc_sumw2[{'process': grouping_sumw2[proc]}][{'process': sum}].as_hist({}).values(flow=True)[1:] for proc in grouping_sumw2}
     else:
         vals = [h_mc[{'process': proc}].eval({})[()][1:-1] for proc in grouping]
+        sumw2 = [h_mc_sumw2[{'process': grouping_sumw2[proc]}][{'process': sum}].eval({})[()][2:-1] for proc in grouping_sumw2]
         mc_vals = {proc: h_mc[{'process': proc}].as_hist({}).values(flow=True)[1:] for proc in grouping}
+        mc_sumw2 = {proc: h_mc_sumw2[{'process': grouping_sumw2[proc]}][{'process': sum}].as_hist({}).values(flow=True)[2:] for proc in grouping_sumw2}
+
     bins = h_data[{'process': sum}].as_hist({}).axes[var].edges
     bins = np.append(bins, [bins[-1] + (bins[-1] - bins[-2])*0.3])
+
+    #Let's define some useful quantities
+    summed_mc_val = h_mc[{'process': sum}].eval({})[()][1:]
+    summed_mc_sumw2 = h_mc_sumw2[{'process': sum}].eval({})[()][1:]
```

Comment:
Should we use `np.sqrt` here, or do we use the squared values directly elsewhere?

Reply:
We do `np.sqrt(summed_mc_sumw2)` later (about like 40 lines below), there should be two lines reading:

`err_m_stat = summed_mc_val - np.sqrt(summed_mc_sumw2` and similar for `err_p_stat`.